app/services/recent-events.ts

```diff
@@ -171,12 +171,12 @@ const filterName = (key: string): string => {
     filter_subscription_9_months: $t('9 Months'),
     filter_subscription_12_months: $t('12 Months'),
     filter_subscription_minimum_enabled: $t('Min. Months'),
-    primesub: $t('Prime'),
+    primesub: $t('Ultra'),
```

Comment:
all of the replacements in this file actually refer to Twitch Prime subs, so they should remain as Prime

Reply:
Done.